discovery-provider/src/queries/search_config.py

```diff
@@ -0,0 +1,15 @@
+import sqlalchemy
+
+trackTitleWeight = 0.7
+userNameWeight = 0.7
+playlistNameWeight = 0.7
+minSearchSimilarity = 0.1
+
+
+def set_search_similarity(session):
+    """
+    Sets the search similarity threshold to be used by % operator in queries.
+    """
+    session.execute(sqlalchemy.text(
+        f"SET pg_trgm.similarity_threshold = {minSearchSimilarity}"
+    ))
```

Comment:
old function call is `select set_limit()` did you replace it with `pg_trgm.similarity_threshold` because set_limit is deprecated?
https://www.postgresql.org/docs/11/pgtrgm.html

can you throw a comment/link in here? i failed to document a lot of this deep pg stuff and it's painful to go look it up each time lol

Reply:
yep, looks like after PG 9.6, similarity_threshold replaced set_limit which is just an alias now. I updated the docstring.

Reply:
👍